File: sample/app/src/main/java/me/example/sauce/databinding/MainActivity.java
package me.example.sauce.databinding;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import android.view.View;

import com.example.sauce.databinding.R;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        findViewById(R.id.bt).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Main2Activity.start(MainActivity.this);
            }
        });
    }
}

File: sample/app/src/main/java/me/example/sauce/databinding/MainActivity.kt
package me.example.sauce.databinding

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View

import com.example.sauce.databinding.R
import me.sauce.InjectExtra

class MainActivity : AppCompatActivity() {
    @InjectExtra("222")
     var ee = 0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        findViewById(R.id.bt).setOnClickListener { Main2Activity.start(this@MainActivity) }
    }
}
